recipe/install-rust-src.bat

```diff
@@ -0,0 +1,8 @@
+set MSYSTEM=MINGW%ARCH%
+set MSYS2_PATH_TYPE=inherit
+set CHERE_INVOKING=1
+FOR /F "delims=" %%i in ('cygpath.exe -u "%PREFIX%"') DO set "PREFIX=%%i/Library"
+FOR /F "delims=" %%i in ('cygpath.exe -u "%SRC_DIR%"') DO set "SRC_DIR=%%i"
+copy "%RECIPE_DIR%\install-rust-src.sh" .
```

Comment:
Is this `copy` statement necessary?

Reply:
I just copied these lines from install-rust.bat and it works.  😂